src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -426,7 +430,7 @@ export class SQLFuture extends Disposable implements FutureInternal {
 			channel: 'shell',
 			type: 'execute_reply',
 			content: {
-				status: 'ok',
+				status: this._errorOccurred && this._stopOnError ? 'error' : 'ok',
```

Comment:
Can we also handle cancel support (can be in separate PR)? We should be sending 'abort' in that case? This is good future proofing, since even if "Run all cells" handles cancel case right now, this protects us in scenarios like background cell execution where we might not have same implementation.

Reply:
#7925